src/misc/meson.build

```diff
@@ -1,5 +1,6 @@
 libmisc_sources = [
   'cross.cpp',
+  'ethernet.cpp',
```

Comment:
Good :) Please add it to VS solution as well (in this commit).

Reply:
Part of solution and filters.